ujcatapi/main.py

```diff
@@ -81,12 +82,22 @@ def add_exception_handlers(app: FastAPI) -> None:
 
 
 def add_middlewares(app: FastAPI) -> None:
-    @app.middleware("http")
-    async def replace_content_type_header(request: Request, call_next: Callable) -> Response:
-        response = await call_next(request)
-        if response.headers.get("content-type") == "application/json":
-            response.headers["content-type"] = "application/json; charset=utf-8"
-        return response
```

Comment:
[question] Do or don't we need this part, and why?

Reply:
Yes, I came to know this today. With this CORS error got solved, but CI fails. I will check this now.

Reply:
I do not believe the removed part led to CI failing, but this middleware is needed so the correct headers are set.
That way the client know it is dealing with JSON instead of guessing it.